src/components/login/input.tsx

```diff
@@ -0,0 +1,45 @@
+import { useRef, useState } from "react";
+
+export default function Input({ props }: any) {
+    //FAZER USANDO CONTEXT
+    const [userEmail, setEmail] = useState('');
+    const [userPassword, setPassword] = useState('');
+
+    const MAX_LENGTH_EMAIL = 30;
+    const MAX_LENGTH_PASS = 50;
+    const MIN_LENGTH_PASS = 6;
+
+    function validateInput(value: string, type: string) {
+        if(type === 'email') { 
+            const validHas: RegExp =  /^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/;
+            validHas.test(value) ? console.log("ok") : console.log("email invalido");
+            
+        }
+        else if (value.length < MIN_LENGTH_PASS) {
+            console.log(`campo vermelho enquanto menos de 6 caracteres` );
+        }
+    }
+
+    //pegando email e senha
+    function handleInput(value: string, type: string) {        
```

Comment:
Aqui também pode utilizar um nome mais explícito

exemplo:
```suggestion
    function getEmailAndPassword(value: string, type: string) {        
```

Reply:
Ajustada.
Agora é uma função de validação.